presto-native-execution/CMakeLists.txt

```diff
@@ -60,6 +60,9 @@ option(PRESTO_ENABLE_TESTING "Enable tests" ON)
 
 option(PRESTO_ENABLE_JWT "Enable JWT (JSON Web Token) authentication" OFF)
 
+option(PRESTO_ENABLE_PROMETHEUS_REPORTER
```

Comment:
It might be helpful to expose this as env var so it can be enabled building by 
```
export PRESTO_ENABLE_PROMETHEUS_REPORTER=ON
```

This change would be in the Makefile. The PRESTO_ENABLE_JWT does this.
It would make injecting this easier from a build script.